specification/serialconsole/resource-manager/Microsoft.SerialConsole/stable/2018-05-01/serialconsole.json

```diff
@@ -169,111 +163,141 @@
             "$ref": "#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "#/parameters/default"
+            "$ref": "#/parameters/defaultParameter"
           }
         ],
         "responses": {
           "200": {
             "description": "OK - Serial Console disabled",
             "schema": {
-              "$ref": "#/definitions/SetDisabledResult"
+              "$ref": "#/definitions/EnableSerialConsoleResult"
             }
           },
-          "default": {
-            "description": "Error response describing why the operation failed.",
+          "404": {
+            "description": "Subscription not found - returns a JSON object with error details",
             "schema": {
-              "$ref": "#/definitions/DeploymentValidateResult"
+              "$ref": "#/definitions/GetSerialConsoleSubscriptionNotFound"
             }
           }
         },
         "x-ms-examples": {
-          "Enable console successfully": {
-            "$ref": "./examples/DisableConsoleExamples.json"
-          },
-          "Disable console with error": {
-            "$ref": "./examples/DisableConsoleExamples.json"
+          "Enable Serial Console for a subscription": {
+            "$ref": "./examples/EnableConsoleExamples.json"
           }
         }
       }
     }
   },
   "definitions": {
-    "SerialConsoleGetResult": {
+    "GetSerialConsoleDisabledResult": {
```

Comment:
could we just call it SerialConsoleStatus?

if it's called GetSerialConsoleDisabledResult, the structure with such name will appear in SDK with just one boolean parameter called "disabled"